core/jobs/src/main/resources/collectors/accessibility/htmlcs.min.js

```diff
@@ -16,50 +16,645 @@
  * limitations under the License.
  */
 /*
-* based on  https://github.com/yargalot/AccessSniff/blob/master/libs/dist/HTMLCS.min.js
+* based on https://github.com/yargalot/AccessSniff version 3.2.0
 * changed scope of variable to window
 * added HTMLCS.getMessagesJSON()
 */
+(function (root, factory) {
```

Comment:
It does not look like minified file.
But I see that previous versions was not minified also (and was named `...min.js`).